client/src/components/auth/LoginForm.jsx

```diff
@@ -10,33 +10,48 @@ import { Button, ControlLabel, Form, FormGroup, FormControl } from 'react-bootst
 class LoginForm extends React.Component {
   constructor(props) {
     super(props);
+
     this.state = {
       username: '',
       password: ''
     };
   }
 
+  handleChange(e) {
+    this.setState({[e.target.id]: e.target.value});
+  }
+
   handleSubmit(e) {
     e.preventDefault();
     this.props.action(this.state);
   }
 
-  handleChange(e) {
-    this.setState({[e.target.name]: e.target.value});
-  }
-
   render () {
     return (
       <Grid>
         <Row>
-        Username 
-        <input name="username" placeholder="Bubbles" onChange={this.handleChange.bind(this)}/>
-        Password:
-        <input name="password" type="password" placeholder="" onChange={this.handleChange.bind(this)}/>
-        <button onClick={this.handleSubmit.bind(this)}>{this.props.controlLabel}</button>
+          <Form horizontal onSubmit={this.handleSubmit.bind(this)}>
```

Comment:
This could be refactored to use form action and method attributes. This class might not need to have state at all.